scipy/linalg/matfuncs.py

```diff
@@ -668,3 +669,60 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def khatri_rao(a, b):
+    r"""
+    Khatri-rao product
+    A column-wise Kronecker product of two matrices
+
+    Parameters
+    ----------
+    a:  (n, k) arraylike
+        Input array
+    b:  (m, k) arraylike
+        Input array
+
+    Returns
+    -------
+    c:  (n*m, k) ndarray
+        Khatri-rao product of `a` and `b`.
+
+    Notes
+    -----
+    The mathematical definition of the Khatri-Rao product is:
+    .. math::
+
+        (A_{ij}  \bigotimes B_{ij})_{ij}
+
+    See Also
+    --------
+        scipy.linalg.kron : Kronecker product
```

Comment:
This line is misaligned; needs to line up with the beginning of `See`

Reply:
Also, use `kron : ` rather than `scipy.linalg.kron :`. It's in the same module, so no full module name needed

Reply:
ok